long-term-storage-aws.md

```diff
@@ -16,9 +36,9 @@ config:
   endpoint: "s3.amazonaws.com"
   region: "us-east-1"
   access_key: "AKIAXW6UVLRRTDSCCU4D"
```

Comment:
Lets replace the access key value with something similar to what was done for the secret key value.